lib/daru/view/adapters/googlecharts/display.rb

```diff
@@ -35,14 +38,13 @@ def show_script(dom=SecureRandom.uuid, options={})
     #   Chart should be rendered in
     # @return [String] js code to render the chart with script tag
     def show_script_with_script_tag(dom=SecureRandom.uuid)
-      # if it is data table and importing data from spreadsheet
-      if is_a?(GoogleVisualr::DataTable) && data.is_a?(String)
```

Comment:
How your changes is handling this if condition?

Reply:
I have kept the same method name for both table and chart, and now have put that method in the common module.

Reply:
@Prakriti-nith , I don't think your replay is relevant . Now are you using the method `to_js_full_script_spreadsheet`  ?